package.json

```diff
@@ -62,6 +62,7 @@
 		"onCommand:cosmosDB.deleteDocDBDocument",
```

Comment:
Where are the tests?  I see neither CTI testcases nor integration/unit tests.  Ideally there would be a simple testcase or two for CTI and some integration tests.